recipes-apps/ovmenu-ng-skripts/files/update-maps.sh

```diff
@@ -22,3 +22,4 @@ fi
 
 
 umount /dev/sda1
+echo "Done !!"
```

Comment:
Please make it one exclamation mark and remove the space before it, like this: `Done!`. It is considered an error to put space before punctuation in English (and all the other languages I know for that matter). 

Here and in other places